rust_icu_sys/bindgen/run_bindgen.sh

```diff
@@ -136,8 +140,18 @@ readonly BINDGEN_ALLOWLIST_FUNCTIONS=(
 )
 
 function check_requirements() {
-  icu-config --version &> /dev/null || \
-    (echo "The generator requires icu-config to be in PATH; see README.md"; exit 1)
+  if [[ 1 -eq "${USE_ICU_CONFIG:-1}" ]]; then
```

Comment:
I'm not sure that this change in bindgen is needed at all. Can we take another look at what the issue seems to be?

Can you run bindgen in the faulty configuration and file a bug with the *entire* log file?